book/http_fundamentals.rst

```diff
@@ -41,9 +41,6 @@ Symfony is built from the ground up around that reality. Whether you realize
 it or not, HTTP is something you use every day. With Symfony, you'll learn
 how to master it.
 
-.. index::
-   single: HTTP; Request-response paradigm
```

Comment:
Question? Is this used anywhere? They are placed so confusingly. They are just above some  (sub)titles.

Sorry for asking.. When I correct the mistakes and undo some thing I just normally push o do I have to make another pull request? My first time contributing, may I be forgiven :)